code/AssetLib/Blender/BlenderScene.h

```diff
@@ -741,8 +740,7 @@ struct Base : ElemBase {
     std::shared_ptr<Object> object WARN;
 
     Base() :
-            ElemBase(), prev(nullptr), next(), object() {
-        // empty
```

Comment:
base class default c’tors are always called; `std::shared_ptr` is always default-initialized